tests/phpunit/tests/rest-api/rest-categories-controller.php

```diff
@@ -117,13 +117,15 @@ public function test_context_param() {
 		$request  = new WP_REST_Request( 'OPTIONS', '/wp/v2/categories' );
 		$response = rest_get_server()->dispatch( $request );
 		$data     = $response->get_data();
+		$this->assertSameSets( array( 'v1' => true ), $data['endpoints'][0]['allow_batch'] );
```

Comment:
```suggestion
		$this->assertSame( array( 'v1' => true ), $data['endpoints'][0]['allow_batch'] );
```

`assertSameSets` is only for things that are numeric arrays. If you use them for keyed arrays, the keys get lost when the values are sorted.